pyproject.toml

```diff
@@ -30,6 +30,12 @@ classifiers = [
 requires-python = ">=3.9"
 dynamic = ["version", "dependencies", "optional-dependencies"]
 
+[project.optional-dependencies]
+monitoring = [
+    "openai",
+    "tiktoken"
```

Comment:
can we update the notebook with the right pip install pacakge?

Reply:
done